editor/README.md

```diff
@@ -11,9 +11,10 @@ Editor overview [here](README_SYSTEMS_OVERVIEW.md).
 * Windows - [Setup instructions](README_SETUP_WINDOWS.md)
 * Linux - [Setup instructions](README_SETUP_LINUX.md)
 
-If you want to use the Visual Studio Code IDE (recommended), read the [VSCode setup guide](README_VSCODE.md).
+If you want to use the Cursive plugin with IntelliJ IDEA (recommended), read the [Cursive setup guide](README_CURSIVE.md).
+
+If you want to use the Calva plugin with Visual Studio Code, read the [VSCode setup guide](README_VSCODE.md).
```

Comment:
I changed the recommended IDE to Cursive since all of us that work primarily with the editor code base use it.